go/vt/topo/tablet.go

```diff
@@ -282,21 +284,31 @@ func (ts *Server) GetTabletAliasesByCell(ctx context.Context, cell string) ([]*t
 	return result, nil
 }
 
+// GetTabletsByCellOptions controls the behavior of
+// Server.FindAllShardsInKeyspace.
+type GetTabletsByCellOptions struct {
+	// Concurrency controls the maximum number of concurrent calls to GetTablet.
+	// For backwards compatibility, concurrency of 0 is considered unlimited.
```

Comment:
I don't think this is right anymore, is it? I say that because we currently only use the value if it's > 0.

Reply:
Yeah.. this has gone through 3 different implementations at this point, and somewhere in there we lost the infinite concurrency. I'm inclined to think 32 is fine as a default and we don't need to worry about infinite concurrency at all 🤷